src/main/java/ch/njol/skript/hooks/chat/expressions/ExprPrefixSuffix.java

```diff
@@ -56,44 +60,57 @@ public class ExprPrefixSuffix extends SimplePropertyExpression<Player, String> {
 	private boolean prefix;
 	
 	@Override
-	public boolean init(final Expression<?>[] exprs, final int matchedPattern, final Kleenean isDelayed, final ParseResult parseResult) {
+	public boolean init(Expression<?>[] exprs, int matchedPattern, Kleenean isDelayed, ParseResult parseResult) {
 		prefix = parseResult.mark == 1;
 		return super.init(exprs, matchedPattern, isDelayed, parseResult);
 	}
 	
 	@Override
-	public String convert(final Player p) {
-		return Utils.replaceChatStyles(prefix ? "" + VaultHook.chat.getPlayerPrefix(p) : "" + VaultHook.chat.getPlayerSuffix(p));
-	}
-	
-	@Override
-	protected String getPropertyName() {
-		return prefix ? "prefix" : "suffix";
+	public String convert(Player player) {
+		return Utils.replaceChatStyles(prefix ? "" + VaultHook.chat.getPlayerPrefix(player) : "" + VaultHook.chat.getPlayerSuffix(player));
 	}
-	
-	@Override
-	public Class<? extends String> getReturnType() {
-		return String.class;
-	}
-	
+
 	@Override
 	@Nullable
 	public Class<?>[] acceptChange(final ChangeMode mode) {
 		if (mode == ChangeMode.SET)
 			return new Class[] {String.class};
 		return null;
 	}
```

Comment:
Did you test this? You never updated the `acceptChange` method. The reset/delete modes aren't valid